src/components/AnswersTable.tsx

```diff
@@ -0,0 +1,95 @@
+import { defaultAnswerType } from "../constants";
+import { Answer as AnswerType, AnswerDisplayType, ID, Question } from "src/types";
+import { Answer } from "./Answer";
+import { Feedback } from "./Feedback";
+
+export interface AnswersTableProps {
+  question: Question;
+  type?: AnswerDisplayType;
+  answer_id?: ID;
+  correct_answer_id?: ID;
+  incorrectAnswerId?: ID;
+  answerIdOrder?: ID[],
+  feedback_html: string;
+  correct_answer_feedback_html?: string;
+  answered_count?: number;
+  show_all_feedback?: boolean;
+  onChangeAnswer: () => void;
+  hideAnswers: boolean;
+  hasCorrectAnswer?: boolean;
+  onChangeAttempt?: () => void;
+  choicesEnabled?: boolean;
+  onKeyPress?: () => void;
+  contentRenderer?: JSX.Element;
+}
+
+export const AnswersTable = (props: AnswersTableProps) => {
+  let idCounter = 0;
+
+  const {
+    question, hideAnswers, type = defaultAnswerType, answered_count, choicesEnabled, correct_answer_id,
+    incorrectAnswerId, answer_id, feedback_html, correct_answer_feedback_html,
+    show_all_feedback = false, hasCorrectAnswer, onChangeAnswer, onKeyPress, answerIdOrder
+  } = props;
+  if (hideAnswers) { return null; }
+
+  const { id } = question;
+
+  const feedback: { index: number, html: string }[] = [];
+  const instructions = undefined;
```

Comment:
what's the deal with this?

Reply:
Good catch -- I intended to convert this to a prop that the tutor-side can pass, because I don't think we'll be needing it. (and I don't recall ever seeing it render in tutor, I think it could be a legacy thing?) Updated: https://github.com/openstax/assignment-components/pull/2/commits/7bb20a93b80e4613a83d690c23bc6afe0f091008